docs/interfaces/internal_.Eip1193Provider.md

```diff
@@ -0,0 +1,29 @@
+[iexec](../README.md) / [Exports](../modules.md) / [<internal\>](../modules/internal_.md) / Eip1193Provider
+
+# Interface: Eip1193Provider
+
+[<internal>](../modules/internal_.md).Eip1193Provider
+
+## Table of contents
+
+### Methods
+
+- [request](internal_.Eip1193Provider.md#request)
+
+## Methods
+
+### request
```

Comment:
```suggestion
### Request
```

Reply:
This documentation is auto-generated from TS typing, here is a method name (camel cased) defined in ethers lib. IMO the current state is fine.